### PR TITLE
Upgrade to Terraform 0.13

### LINKED ISSUE #788: Upgrade to Terraform 0.13

### PR DESCRIPTION
**Pull request by invidian**
~NOTE: Includes commits from #860.~

~NOTE: Includes changes from #1027~

Closes #788

Signed-off-by: Mateusz Gozdek <mateusz@kinvolk.io>

### CODE REVIEW

**Comment by invidian** [comment]
Rebased to include #1021, so CI can pass.

File: pkg/terraform/executor_test.go [hunk 1]
```diff
@@ -17,21 +17,50 @@ func TestVersionConstraint(t *testing.T) {
...
+		"not JSON": {
 			output: `Terraform v0.13.11
 
 Your version of Terraform is out of date! The latest version
 is 0.13.3. You can update by downloading from https://www.terraform.io/downloads.html`,
+			expectError: true,
+		},
+		"empty version field": {
+			output: `{
+	"terraform_version": "",
```
Comment (knrt10) [inline pkg/terraform/executor_test.go:57]: Small lint fix, spaces can be removed I guess

**Review by knrt10 — COMMENTED** [review]
Rest looks great

**Comment by invidian** [comment]
Ah, forgot to mention. This PR includes #1027, so #1027 should get merged first.

**Review by knrt10 — APPROVED** [review]
looks great

**Comment by surajssd** [comment]
The code looks ok to me, @invidian could you rebase and trigger the CI once more?

**Comment by invidian** [comment]
@surajssd this is already based on latest master. I try to restart AKS job now..

**Comment by invidian** [comment]
Well, re-pushed anyway, as CI seems to got stuck somehow

**Comment by knrt10** [comment]
Restarted failed aks and packetARM pipelines

**Review by knrt10 — APPROVED** [review]

**Review by knrt10 — APPROVED** [review]

**Pull request merged**